cfme/web_ui/menu.py

```diff
@@ -22,7 +22,8 @@ def get_current_toplevel_name():
 def _tree_func_with_grid(*args):
     def f():
         accordion.tree(*args)
-        toolbar.select('Grid View')
+        if toolbar.exists('Grid View'):
+            toolbar.select('Grid View')
```

Comment:
It seems to be caused by https://bugzilla.redhat.com/show_bug.cgi?id=1289190, so its indeed a bug and it seems there is no easy workaround for it